<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>538</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Formalización de creditos</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>INFORMACION CLIENTE </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CEDULA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCedula">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>NOMBRE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>GENERO:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGenero">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MASCULINO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FEMENINO</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TIPO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>MONTO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAZO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TASA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTRO DE OPERACION DE CREDITO </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonto">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>SECTOR:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSector">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PUBLICO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRIVADO</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxTipoCredito">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>VIVIENDA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CONSUMO </string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtTasa">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPlazo">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>3 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>48 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>96 </string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBoxCargos">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CARGOS ADMINISTRATIVOS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TblInformacion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Cédula</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Genero</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sector</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo de crédito</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Monto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tasa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Plazo en meses</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>421</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="BtnGenerar">
      <property name="text">
       <string>GENERAR COTIZACION</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnModificar">
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnEliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
